mscore/prefsdialog.ui

```diff
@@ -688,7 +688,7 @@
            </widget>
           </item>
           <item row="1" column="2">
-           <widget class="QLabel" name="iconHeigthLabel">
+           <widget class="QLabel" name="iconHeightLabel">
```

Comment:
This is a code change, but OK as long as that name isn't referenced elsewhere

Reply:
> This looks to be a unique source typo as well.

Indeed, that's why i made a comment on it (but you beat me to it :smiley:)

Reply:
@MarcSabatella here is a source change :point_up_2:

Reply:
Yes, but labels are almost never referenced by the code.  If you didn't do a search on any such strings already, do so, but I'm guessing there won't be an issue.

Reply:
I did and it is not

Reply:
I did as well. So that does it then, squash + rebase time ?

Reply:
yes